{
    "name": "invalid_in_configure",
    "description": "reports an invalid configuration, in the computation configure function",
    "computations": {
        "(client)": {
            "messages": {
                "testcomp": "*"
            }
        }, 
        "testcomp": {
            "dso": "libarras4_testcomputation.so", 
            "entry": "yes", 
            "forward":true,
            "errorInConfig":"invalid",
            "requirements": {
                "computationAPI": "4.x",
                "rez_packages": "arras4_test-4",
                "resources": {
                    "cores": 1.0,
                    "memoryMB": 16384
                }
            },
            "messages": {
                "(client)": "*" 
            }
        }
    }
}
